health/micro-ui-react19/web/packages/css/src/pages/employee/healthdss.scss

```
@@ -887,10 +897,10 @@
       border: 0.063rem solid;
```

Stylelint (17.7.0)

[error] Duplicate property "border" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)